fw/preamp/src/audio.c

```diff
@@ -196,3 +202,32 @@ InputType getSourceAD(size_t src) {
   }
   return IT_ANALOG;
 }
+
+void initAudio() {
+  // Initialize each zone's audio state (does not write to volume control ICs)
+  for (size_t zone = 0; zone < NUM_ZONES; zone++) {
+    mute(zone, true);
+    setZoneSource(zone, DEFAULT_SOURCE);
+  }
+
+  // Initialize each source's analog/digital state
+  for (size_t src = 0; src < NUM_SRCS; src++) {
+    setSourceAD(src, IT_DIGITAL);
```

Comment:
```suggestion
    // Set all sources to Digitial initially since the digital input is by default nothing.
    // This avoids any unwanted playback.
    // Also, expanders need to be set to digital to get passed through audio (this is also expander fw).
    setSourceAD(src, IT_DIGITAL);
```